cp-connector/pkg/controlplane/logforwarder_test.go

```diff
@@ -0,0 +1,63 @@
+package controlplane
+
+import (
+	"testing"
+
+	"github.com/keptn/go-utils/pkg/api/models"
+	"github.com/keptn/go-utils/pkg/common/strutils"
+	keptnv2 "github.com/keptn/go-utils/pkg/lib/v0_2_0"
+	"github.com/keptn/keptn/cp-connector/pkg/controlplane/fake"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLogForwarderNoForward(t *testing.T) {
```

Comment:
let's also add some checks whether `logHandler.Log()` was called or not to each of these test cases?

Reply:
adapted, thank you